app/src/main/java/org/simple/clinic/summary/PatientSummaryScreen.kt

```diff
@@ -272,12 +280,12 @@ class PatientSummaryScreen(context: Context, attrs: AttributeSet) : RelativeLayo
 
   fun showBloodPressureEntrySheet(patientUuid: UUID) {
     val intent = BloodPressureEntrySheet.intentForNewBp(context, patientUuid)
-    activity.startActivity(intent)
+    activity.startActivityForResult(intent, REQCODE_BP_ENTRY)
   }
 
   fun showBloodPressureUpdateSheet(bloodPressureMeasurementUuid: UUID) {
     val intent = BloodPressureEntrySheet.intentForUpdateBp(context, bloodPressureMeasurementUuid)
-    activity.startActivity(intent)
+    activity.startActivityForResult(intent, REQCODE_BP_ENTRY)
```

Comment:
This will also trigger the dialog when an older BP is updated. Is this expected?

Reply:
Hmmm… good catch. Editing of a BP cannot be considered as a "patient visit". I'll remove this.